R/robustbayesianmetaanalysis.R

```diff
@@ -739,38 +743,38 @@ RobustBayesianMetaAnalysis <- function(jaspResults, dataset, options, state = NU
         "omega_alternative" = priors[["omega"]]
       )
       
-      if (length(temp_priors) == 0)
+      if (length(tempPriors) == 0)
         next
       
       # generate the actual plots
-      for (i in 1:length(temp_priors)) {
+      for (i in 1:length(tempPriors)) {
```

Comment:
Don't you want to seq_along here? In my experience that's a bit more robst.

Reply:
the analysis can't run if the length is lower than 3, but I can add it

Reply:
done